airflow/providers/amazon/aws/operators/s3_file_transform.py

```diff
@@ -16,10 +16,12 @@
 # specific language governing permissions and limitations
 # under the License.
 
+# pylint: disable=R0913
```

Comment:
It would be better if you could add it to the actual place where it occurs - probably line 88 or 100 `# pylint: disable=too-many-arguments`